server/sonar-web/src/main/less/components/page.less

```diff
@@ -123,6 +123,10 @@
   }
 }
 
+.page-footer-with-sidebar {
+  padding-left: ~"calc(50vw - 370px + 10px)" !important;
```

Comment:
I also suggest to limit its max-width to match the width of the content.

Reply:
good idea!